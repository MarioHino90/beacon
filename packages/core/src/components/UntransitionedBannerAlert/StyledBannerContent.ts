import styled from "styled-components";

export const StyledBannerContent = styled.div`
  display: flex;
  flex-direction: column;
  width: 100%;
`;
