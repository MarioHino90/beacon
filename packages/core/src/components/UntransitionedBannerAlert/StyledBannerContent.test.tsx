import React from "react";
import { shallow } from "enzyme";

import { StyledBannerContent } from "./StyledBannerContent";

describe("StyledBannerContent", () => {
  it("should render correctly", () => {
    const wrapper = shallow(
      <StyledBannerContent>
        <div>Banner Child</div>
      </StyledBannerContent>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
