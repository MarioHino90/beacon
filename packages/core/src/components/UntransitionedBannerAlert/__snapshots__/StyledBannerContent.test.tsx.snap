// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledBannerContent should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
}

<div
  className="c0"
>
  <div>
    Banner Child
  </div>
</div>
`;
